Composer/packages/lib/code-editor/src/BaseEditor.tsx

```diff
@@ -30,7 +30,7 @@ const defaultOptions = {
   lineDecorationsWidth: 10,
   lineNumbersMinChars: 3,
   glyphMargin: false,
-  folding: false,
+  folding: true,
```

Comment:
Let's only override this setting in the LU and LG editors.

Reply:
changed the lg folding option to LGEditor